app/src/main/java/com/library/LendingSystem.java

```diff
@@ -0,0 +1,96 @@
+package com.library;
+
+import java.util.*;
+
+class LendingSystem {
+    private Map<Book, Patron> borrowedBooks = new HashMap<>();
```

Comment:
The lending system should not store internal details of patrons and books, instead you can link them through their unique identifiers like isbn, userid etc.

User, as an individual entity should be able to store all the users in it's own class and database and not tightly coupled with lending system.

Reply:
I see you are already maintaining the list in the corresponding inventory classes for books and patrons. You can get the data from those lists with the id's.